src/main/java/service/CreateReport.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.Fruit;
+
+public interface CreateReport {
+    String createReport(List<Fruit> fruits);
```

Comment:
```suggestion
    String createReport();
```
go to Storage inside implementation